Grades.java

```diff
@@ -13,6 +15,55 @@
 public class Grades {
 
     public static void main(String[] args) {
-        // enter your code here :)
+
+        Scanner input = new Scanner(System.in);
+        System.out.printf("Please enter the number of students (up to 127 allowed): ");
+        byte numOfStudents = input.nextByte();
+
+        if (numOfStudents <= 0) {
+            System.out.println("The number of students must be positive.");
+            return;
+        }
+
+        byte studentsArray[] = new byte[numOfStudents];
+        short grade = 0;
+        byte indexHighest = 0;
+        byte indexLowest = 0;
+        byte highest = 0;
+        byte lowest = 100;
```

Comment:
Do you really need the highest and the lowest VALUES ?
Index is enough.. isn't it ?

Reply:
@ameedf Getting the highest and lowest values was a part of the assignment..

Reply:
indeed.. but my question is: isn't knowing the index is enough ? why do you have to keep (in memory) these variables ? isn't it a "waste" of memory ? 
I mean whenever you wrote 'highest' you could've wrote 'myArray[ indexHighest ]' :)

Reply:
You are totally correct! I could do that :-)